packages/makefile/action.yaml

```
@@ -129,7 +129,7 @@ runs:
         yq-version: v4.11.2
```

```suggestion
        yq-version: v4.30.6
```

Ini sekalian aja gak mas @adeherysh ?